templates/docs/utilities/vertical-spacing.md

```diff
@@ -8,6 +8,24 @@ context:
 
 <hr>
 
+A preferable way to control bottom spacing of an element is using Vanilla pre-defined classes: `.u-sv--shallow`, `.u-sv--regular` and `.u-sv--deep`:
+
+| Class            | Value                                    |
+| ---------------- | ---------------------------------------- |
+| `.u-sv--shallow` | Increases space by 3 baseline grid units |
```

Comment:
I think it would be more useful to define the "value" in rems rather than spacing units. Spacing units make sense in table below (where the number in class name corresponds to number of units directly)

So I think this table here may follow the Figma file in the structure:
<img width="777" alt="image" src="https://user-images.githubusercontent.com/83575/228556826-88c23d7b-7f32-47ad-9b95-436c260f81a9.png">

So let's have "Class" (same as current), "Height" (expressed as rem), and "Use" column that defines when to use each size. I think we can just use the content from Figma (but edit out the parts that refer to other sections of Figma file).

Reply:
@bartaz , but the `$sp-unit-ratio` is available for overriding, and if somebody does it, these sizes will be changed automatically. So absolute values in rems might be not accurate.

Reply:
@yurii-vasyliev Good point. I don't think we expect someone to do it, and it may have some breaking consequences if someone does, but still we may take that into account.

On our [spacing documentation](https://vanillaframework.io/docs/settings/spacing-settings) page we mention spacing unit mulitplier and default value, maybe we can do something similar here:

<img width="1129" alt="image" src="https://user-images.githubusercontent.com/83575/228572318-d4e306e7-ed60-468d-bc11-101edeef65c7.png">

So, keep the value expressed by sp-unit, but add a column with default value in rem.